src/App.tsx

```diff
@@ -1,61 +1,27 @@
 import './App.scss';
+import { useState, FC } from 'react';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { AddUser } from './components/AddUser/AddUser';
+import { Todo } from './types/types';
+import { getUserById } from './utils/getUserByID';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: FC = () => {
+  const initialTodos: Todo[] = todosFromServer.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
```

Comment:
[CODE KNOWLEDGE] - If you are using a non-mutating array method, you **don't** need to create a copy of the array. In your case, you're using the `map` method which is non-mutating, so no need to create a copy of `todo` with `{...todo}`. You can directly add the `user` property to `todo`.